posthog/conftest.py

```diff
@@ -7,7 +7,7 @@
 from posthog.test.base import PostHogTestCase, run_clickhouse_statement_in_parallel
 
 
-def create_clickhouse_tables(num_tables: int):
```

Comment:
This change (and the other test changes to/around `create_clickhouse_tables`) wasn't actually necessary: I thought I as going to need to do more invasive fixture surgery than just the `dag_tests/conftest.py` edit and found the coupling to `django_db_setup` confusing so I simplified the logic here before realizing I wouldn't need to call this function directly.